pkg/skaffold/config/options_test.go

```diff
@@ -77,11 +92,12 @@ func TestLabels(t *testing.T) {
 				},
 			},
 			expectedLabels: map[string]string{
-				"cleanup": "true",
-				"one":     "first",
-				"two":     "second",
-				"three":   "",
-				"four":    "",
+				"cleanup":          "true",
```

Comment:
why  cleanup and skaffold-cleanup both exist here?

Reply:
If I understand correctly, in #2002, it's question of keeping both for sometimes to make sure not to break potential use from skaffold users.
If this is excessive precaution, we might just replace the existing ones.

Reply:
ah understood. this is for backward compatibility.  However these are labels that skaffold uses internally to track pods that needed to cleaned-up. Users should not have to care about these labels.
So. removing them should be fine.